README.md

```diff
@@ -16,24 +16,26 @@ The app launches a window displaying a map.
 2. Choose the java-gradle-starter-project directory and click _OK_.
 3. Select _File > Project Structure..._ and ensure that the Project SDK and language level are set to use Java 11.
 4. Open the Gradle view with _View > Tool Windows > Gradle_.
-5. Refresh the Gradle project by clicking the _Refresh all Gradle projects_ button in the Gradle view toolbar. This will cause the ArcGIS Runtime SDK Gradle plugin to download the native libraries.
-6. In the Gradle view, double-click `run` under _Tasks > application_ to run the app.
+5. Refresh the Gradle project by clicking the _Refresh all Gradle projects_ button in the Gradle view toolbar.
+6. In the Gradle view, double-click `run` under _Tasks > application_ to run the app. This task will cause the native libraries to be copied to $USER_HOME/.arcgis before the sample is run.
```

Comment:
The run task triggers the copyNatives task, which does indeed copy native libraries to $USER_HOME/.arcgis. I watched it happen!